style.css

```
@@ -0,0 +1,18 @@
+html {
```

Optional
I think its a great practice to always remember to add the CSS universal selector
```suggestion
* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}
```
This is useful when you want to set a style for of all the elements of an HTML page or for all of the elements within an element of an HTML page.